src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/CBApplicationBase.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    /// <summary>
+    /// The class for supporting multi-components based application model.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class CBApplicationBase : Application
```

Comment:
X DO NOT use abbreviations or contractions as part of identifier names.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions

```suggestion
    public class ComponentBasedApplication : Application
```

Reply:
It is a base class of other applications, therefore I think ComponentBasedApplicationBase is a proper name.
I changed it into ComponentBasedApplicationBase